src/components/Footer.tsx

```diff
@@ -0,0 +1,101 @@
+import classNames from 'classnames';
+import React, { useContext, useEffect, useState } from 'react';
+import { GlobalContex } from '../TodoContext';
+import { Filter } from '../types/Filter';
+import { TodoErrors } from '../types/TodoErrors';
+import { Todo } from '../types/Todo';
+
+export const Footer: React.FC = () => {
+  const {
+    todos,
+    setTodos,
+    filter,
+    setFilter,
+    deleteTodoItem,
+    setIsCompletedRemoving,
+    setError,
+    setIsTitleOnFocus,
+  } = useContext(GlobalContex);
+
+  const [deletedTodos, setDeletedTodos] = useState<Todo[]>([]);
+
+  useEffect(() => {
+    setIsCompletedRemoving(false);
+    setIsTitleOnFocus(true);
+    setTodos(todos.filter(todo => !deletedTodos.includes(todo)));
+  }, [deletedTodos]);
```

Comment:
fix this linter error 
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/108538921/657725b3-a4ca-4770-a79c-3de4e961d229)